src/pal/src/eventprovider/lttng/tpdotnetruntimeprivate.h

```diff
@@ -0,0 +1,1600 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Do not modify the file. This file is generated using the script from <root>/src/inc/genXplatEtw.pl
+//
+
+#include "palrt.h"
+#include "pal.h"
+
+#undef TRACEPOINT_PROVIDER
+#define TRACEPOINT_PROVIDER dotnetruntimeprivate
+
+#undef TRACEPOINT_INCLUDE
+#define TRACEPOINT_INCLUDE "./lttng/tpdotnetruntimeprivate.h"
+
+#if !defined(LTTNG_CORECLR_Hdotnetruntimeprivatedotnetruntimeprivate) || defined(TRACEPOINT_HEADER_MULTI_READ)
+#define LTTNG_CORECLR_Hdotnetruntimeprivate
+
+#include <lttng/tracepoint.h>
+
+
+#define ClrStackWalk_TRACEPOINT_ARGS \
+TP_ARGS(\
+        const unsigned short ,ClrInstanceID,\
+        const unsigned char ,Reserved1,\
+        const unsigned char ,Reserved2,\
+        const unsigned int ,FrameCount,\
+        size_t*,Stack\
+) 
+TRACEPOINT_EVENT_CLASS(
```

Comment:
Is there a reason that you're using the TRACEPOINT_EVENT_CLASS and TRACEPOINT_EVENT_INSTANCE definitions instead of just TRACEPOINT_EVENT?  It doesn't look like the TRACEPOINT_EVENT_CLASSes are getting used multiple times here.